Tasks/Integration/04_ManageEmailSettings.md

```diff
@@ -0,0 +1,132 @@
+[!!User interface Email settings](../UserInterface/02d_EmailSettings.md)
+
+# Manage the email settings
+
+The *EMAIL SETTINGS* tab allows you configure how a user is notified per email when a task is created. It contains a menu on the left with two settings entries: *General settings* and
+*Email sending per event*.
+
+To configure the email general settings, see [Configure the general settings](#configure-the-general-settings). To configure the email notification settings, see [Configure the email sending per event](#configure-the-email-sending-per-event).
+
+> [Info] This tab is only displayed if the *Email* module (1.6.1 version or higher) is installed. 
+
+
+
+## Create a connection in the *Email* module
```

Comment:
Funktioniert das mit *Email* in der Überschrift? Steht nicht irgendwo, dass wir das vermeiden sollen?